de-lib/src/main/java/org/iplantc/de/theme/base/client/collaborators/GroupDisplayStrings.properties

```diff
@@ -1,3 +1,5 @@
+addGroup=Add List
+deleteGroup=Delete List
 deleteGroupConfirm=By deleting the collaborator list {0}, any data that has been shared with this list will immediately be un-shared.  Do you wish to continue deleting the collaborator list?
```

Comment:
My initial thinking was that it would be nice for users to be able to choose to share something with a collaborator list directly rather than with all users in the collaborator list individually. Sharing something with individual users in a collaborator list would still be possible, of course, but if a user is removed from a collaborator list then anything shared with the collaborator list directly would no longer be accessible to the user (unless it was also shared with the user directly or with another group that the user is in. I can see where that could be confusing, though.